google_takeout_parser/parse_json.py

```diff
@@ -25,6 +26,16 @@
 from .time_utils import parse_json_utc_date
 
 
+def _read_json_data(p: Path) -> Any:
+    try:
+        import orjson
```

Comment:
Not sure if you'd prefer to add it to setup.cfg as an optional dependency
Also could use it in other functions in this module, but the rest of data is so small that it would make basically no difference